### ISSUE TITLE
Add a feature to blacklist specific cordinates

### ISSUE THREAD

**Issue by LiterallyThanos** [cabaletta/baritone#1100]
## Describe your suggestion
Give the option to make a list of blacklisted cordinates like the list used with blocks to avoid breaking and this would essentially stop the bot from pathing to this block and wouldnt even recognize its existence 

## Context
for me im using it to path around a map clicking chests but said map has chests that i dont want it to path to because it will never have anything in it

## Final checklist
- [ ] I have not used any OwO's or UwU's in this issue.

**Comment by TexRexTex** [comment]
Yes I think this would be a good idea, maybe like two sets of coordinates to highlight a box area to avoid

**Comment by TexRexTex** [comment]
Also whitelisted coordinates would be good
